src/components/helper-methods.js

```diff
@@ -58,10 +58,17 @@ export default {
     }, {parent: parentProps});
   },
 
-  getCalculatedValues(props, defaultStyles) {
-    const style = Helpers.getStyles(props.style, defaultStyles, "auto", "100%");
-    const colors = Array.isArray(props.colorScale) ?
-      props.colorScale : Style.getColorScale(props.colorScale);
+  getCalculatedValues(props, fallbackProps) {
+    const styleObject = props.theme && props.theme.pie ? props.theme.pie.style
+    : fallbackProps.style;
+    const style = Helpers.getStyles(props.style, styleObject, "auto", "100%");
+    const propsFallbacks = props.colorScale || fallbackProps.colorScale;
```

Comment:
This could be written more clearly as 
```
const theme = props.theme && props.theme.pie;
const colorScale = props.colorScale || (theme && theme.colorScale) || fallbackProps.colorScale;
```
or you could even break it out into a little private function 

```
const theme = props.theme && props.theme.pie;
const getColorScale = () => {
  return theme ? theme.colorScale : fallbackProps.colorScale
};
const colorScale = props.colorScale || getColorScale();
```